src/commands/config/log/set.ts

```diff
@@ -34,28 +34,23 @@ const command: Command = {
 
     await config.updateGuild({ guildId: msg.guildId, logChannel })
 
-    return {
-      messageOptions: {
-        embeds: [
-          composeEmbedMessage(msg, {
-            author: [msg.guild.name, msg.guild.iconURL()],
-            description: `Successfully set <#${logChannel}> as log channel`,
-          }),
-        ],
-      },
-    }
+    const embed = composeEmbedMessage(msg, {
+      author: [msg.guild.name, msg.guild.iconURL()],
+      description: `Successfully set <#${logChannel}> as log channel`,
+    })
```

Comment:
```suggestion
    const embed = getSuccessEmbed({
      msg,
      title: msg.guild.name,
      description: `Successfully set <#${logChannel}> as log channel`,
    })
```
https://github.com/consolelabs/mochi-bot/blob/develop/src/commands/README.md#okfailed-message